docs/modeling-your-data/running-data-models-via-snowplow-bdp/dbt/using-dbt/index.md

```diff
@@ -0,0 +1,179 @@
+      type: bigquery
+      method: oauth
+      project: [GCP project id]
+      dataset: [the name of your dbt dataset] # You can also use "schema" here
+      threads: [1 or more]
+      keyfile: "{{ env_var('GOOGLE_APPLICATION_CREDENTIALS') }}" 
+  target: prod
+```
+
+</TabItem>
+<TabItem value="databricks" label="Databricks">
+
+See [dbt connection setup](https://docs.getdbt.com/reference/warehouse-setups/databricks-setup) for more information.
+
+```yaml
+profile_name:
+  outputs:
+    prod:
+      type: databricks
+      catalog: [optional catalog name, if you are using Unity Catalog, is only available in dbt-databricks>=1.1.1]
+      schema: [schema name]
+      host: [yourorg.databrickshost.com]
+      http_path: [/sql/your/http/path]
+      token: "{{ env_var('DATABRICKS_DATAMODELING_TOKEN') }}" # Personal Access Token (PAT)
+      threads: [1 or more]  # optional, default 1
+  target: prod
+```
+
+</TabItem>
+</Tabs>
+
+### 2. The data modeling configuration
+
+Data models can be configured via the Snowplow BDP Console:
+
+![](images/Screenshot-2021-11-15-at-20.15.28.png)
+
+In the first step, you can provide the data model name, a description, and the owner(s) of the data model that will be alerted in case of failure and finally select the engine that should run your datamodel (DBT in this case):
+
+![](images/data-model-create-step-1.png)
+
+
+Please note that the model name needs to match to the corresponding datamodeling subfolder name in GitHub. The dbt project for the model in the above example would therefore live under:
+
+`your snowplow-pipeline repo > datamodeling > data model name > dbt ...`
+
+In the second step, you can then add the schedule:
+
+![](images/data-model-create-step-2.png)
+
+Please note that the cron schedule is in UTC.
+
+:::caution
+
+You should write your dbt models in such a way that if they fail part way through they can be re-run without issue. All Snowplow models are written in this way and if you write your models following the [custom modules](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-custom-models/) documentation you can take advantage of this.
+
+:::
+
+In the third and last step you can configure the models and variables your `dbt run` call is excecuted with. The `modelArgs` follows a [`--select` flag](https://docs.getdbt.com/reference/node-selection/syntax#shorthand), and `varArgs` follow the [`--vars` flag](https://docs.getdbt.com/docs/build/project-variables#defining-variables-on-the-command-line) in the call:
+
+![](images/data-model-create-step-3.png)
+
+
+Once you are happy with the configuration, you can create the model. The model will be disabled until you enable it:
+
+![](images/Screenshot-2021-11-15-at-20.25.53.png)
+
+Please make sure all your dbt project files are merged to the default branch in GitHub before enabling the model. Note that this tooling does not run a "sync and deploy" process. Any changes merged to the default branch are available immediately.
```

Comment:
> Note that this tooling does not run a "sync and deploy" process. Any changes merged to the default branch are available immediately.

This information is only really relevant if you know how sql-runner jobs work. Maybe just change it to:

```suggestion
Please make sure all your dbt project files are merged to the default branch in GitHub before enabling the model. Any changes merged to the default branch are available immediately.
```